src/claims/models/investor-uniqueness-claim.model.ts

```diff
@@ -0,0 +1,42 @@
+/* istanbul ignore file */
+
+import { ApiProperty } from '@nestjs/swagger';
+import { ClaimType } from '@polymeshassociation/polymesh-sdk/types';
+
+import { ScopeModel } from '~/claims/models/scope.model';
+import { FromEntity } from '~/common/decorators/transformation';
+
+export class InvestorUniquenessClaimModel {
+  @ApiProperty({
+    type: 'string',
+    description: 'Claim type',
+    example: 'InvestorUniqueness',
+  })
+  @FromEntity()
+  readonly type: ClaimType.InvestorUniqueness;
```

Comment:
Do we need to explicitly define the type here? IMO the name suggests itself that the claim type is `InvestorUniqueness`. What do you think @polymath-eric ?

Reply:
I think you are right.

At first I thought the Model name isn't returned to the client, but if we have a unique model for this its redundant. I think we only need `type` if we were reusing the generic claim model.